docs/components/movement-sensor/gps/gps-nmea-rtk-pmtk.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Configure an NTRIP-based RTK GPS with an I2C Connection"
+linkTitle: "gps-nmea-rtk-pmtk"
+weight: 10
+type: "docs"
+description: "Configure an NTRIP-based RTK that uses I2C communication."
+images: ["/icons/components/imu.svg"]
+# SMEs: Susmita
```

Comment:
make sure to add an alias if this is replacing gps-rtk.

Reply:
I already added the alias on the serial one :D Dunno if I should switch it to this one instead? Figured it was either one.

Reply:
nvm I missed that

Reply:
actually - what about rtk-station?

Reply:
@npentrel rtk-station has been removed from rdk. It's not a component.

Reply:
@npentrel rtk-station has been removed from RDK, it's not a component anymore.